src/xcode/ENA/ENA/Source/AppDelegate & Globals/__tests__/CWAHibernationProviderTests.swift

```diff
@@ -24,7 +24,7 @@ final class CWAHibernationProviderTests: CWATestCase {
 		guard let hibernationComparisonDate = Calendar.current.date(from: beforeHibernationStartDateComponents) else {
 			return XCTFail("Expect the hibernation comparison date from the corresponding date components.")
 		}
-		mockTestStore.hibernationComparisonDate = hibernationComparisonDate
+		mockTestStore.hibernationStartDate = hibernationComparisonDate
```

Comment:
We should better use a separate item in the store.

Reply:
we dont need the old flag i renamed it and changed the logic